src/components/victory-candlestick/candle.js

```diff
@@ -20,8 +18,8 @@ export default class Candle extends React.Component {
   }
 
   renderWick() {
-    const width = VictoryCandlestick.defaultProps.width;
-    const padding = VictoryCandlestick.defaultProps.padding;
+    const width = this.props.width;
+    const padding = this.props.padding;
```

Comment:
Does the padding here refer to the padding of the entire chart as it does in VictoryChart, or is this related to padding between individual candlesticks? If so, you should expect to pluck padding off of the style object. Leave the padding prop on `VictoryCandlestick` the same as it is on the other components. It's used to calculate range, and therefore scale, so it needs to match up.